extensions/cornerstone/src/index.tsx

```diff
@@ -140,5 +141,6 @@ export {
   CornerstoneExtensionTypes as Types,
   toolNames,
   getActiveViewportEnabledElement,
+  ImageOverlayViewerTool,
```

Comment:
This just exposes the tool externally to be used.  Otherwise it becomes difficult to add new metadata.